bentoml/_internal/store.py

```diff
@@ -162,3 +163,65 @@ def delete(self, tag: t.Union[str, Tag]) -> None:
                 # otherwise, update the latest version
                 assert new_latest.tag.version is not None
                 self._fs.writetext(_tag.latest_path(), new_latest.tag.version)
+
+    def export_tar(
```

Comment:
Can we not just do `self.get(tag).export("tar://" + path)`?

Reply:
@sauyon in the cli part we are passing a file object instead of path to support both real file and standard out. I think `fs` cannot support this use case?